_includes/search.html

```diff
@@ -0,0 +1,141 @@
+<script src="{{ '/assets/javascript/lunr.js' | relative_url }}"></script>
+
+<script>
+    {% assign counter = 0 %}
+
+    var documents = [{% for page in site.posts %}{
+        "id": {{counter}},
+        "url": "{{ site.baseurl }}{{ page.url }}",
+        "link": "{{ page.link }}",
+        "title": "{{ page.title }}",
+        "author": "{{ page.author }}",
+        "postDate": "{{ page.date | date: '%B %d, %Y'}}",
+        "body": "{{ page.content | markdownify | replace: '.', '. ' | replace: '</h2>', ': ' | replace: '</h3>', ': ' | replace: '</h4>', ': ' | replace: '</p>', ' ' | strip_html | strip_newlines | replace: '  ', ' ' | replace: '"', ' ' }}" 
+        {% assign counter = counter | plus: 1 %}
+    } {% if forloop.last %} {% else %}, {% endif %} {% endfor %}];
+
+    var idx = lunr(function () {
+        this.ref('id')
+        this.field('title')
+        this.field('body')
+        documents.forEach(function (doc) {
+            this.add(doc)
+        }, this)
+    });
+
+    function lunr_search(terms) {
+
+        document.getElementById('lunrsearchresults').innerHTML = '<div></div>';
+
+        if (terms) {
+
+            document.getElementById('lunrsearchresults').innerHTML = document.getElementById('lunrsearchresults').innerHTML + "<p>Search results for '" + terms + "'</p>";
+            document.getElementById('lunrsearchresults').innerHTML = document.getElementById('lunrsearchresults').innerHTML + "<ul class=search-result></ul>";
+
+            mountResultHtml(terms);
+        } else {
+            location.assign(window.location.pathname);
+        }
+
+        return false;
+    }
+
+    function getLunrSearchQuery(query) {
+        const searchTerms = query.split(" ");
+        if(searchTerms.length === 1) {
+            return query;
+        }
+
+        query = "";
+        
+        for(const term of searchTerms) {
+            query += `+${term}`
```

Comment:
Thanks for the updates @gabrielpadilh4!

I think we're missing a space in between search terms here when prefixing the term with a +, i.e., "openshift ejb" would become "+openshift+ejb" and then no results would be found for this query.